supabase/migrations/20230703134143_add-usage-table.sql

```diff
@@ -0,0 +1,30 @@
+create table "public"."usage" (
+    "id" bigint generated by default as identity not null,
+    "org_id" bigint not null,
+    "date" date not null,
```

Comment:
Can this be set with CURRENT_DATE by default? Might make logic elsewhere simpler

![Screenshot from 2023-07-03 15-32-55](https://github.com/Superflows-AI/superflows/assets/33871096/6e2fec3b-c203-40dc-b9cc-2059229ccc4a)